Add malignant cells results section (#234)
AlexsLemonade/ScPCA-manuscript: content/03.results.md

Unlike the consensus cell type annotations, the OpenScPCA project annotations distinguish between normal and malignant cells and contain far fewer uncharacterized cells. 
Indeed, for `SCPCP000004`, the consensus cell type procedure labeled only ~43% of cells, but the OpenScPCA project labeled ~91% of cells, thereby adding substantial value to the data.
<!-- Calculation. TODO: Do we want this added to `scpca-paper-figures` manuscript-numbers folder?
> sum(!(sce$openscpca_celltype_annotation %in% c("Unknown", "openscpca-excluded"))) / ncol(sce)
[1] 0.9095714

> sum(sce$consensus_celltype_annotation != "Unknown") / ncol(sce)
[1] 0.4342133
-->

> We plan to continue expanding the number of projects on the Portal with manually-curated annotations from the OpenScPCA project. 

Reviewer: I agree.

```suggestion
```

Note that the Portal's summary cell type report will include comparisons between annotations made in `scpca-nf` to OpenScPCA annotations for relevant libraries.

> In addition, we included a step in the `scpca-nf` pipeline to run `InferCNV` [@url:https://github.com/broadinstitute/inferCNV] to quantify copy-number alterations across libraries, specifically for libraries with at least 100 normal cells (as determined by consensus cell types) to use as the baseline for CNV inference (Figure {@fig:fig2}A).

Reviewer 2: ```suggestion
In an effort to identify potential malignant cells across all samples in the Portal, we included a step in the `scpca-nf` pipeline to run `InferCNV` [@url:https://github.com/broadinstitute/inferCNV] to quantify copy-number alterations (Figure {@fig:fig2}A).
```
I might add a little more context as to why we are using inferCNV first before saying we added it. I also don't think details like 100 normal cells are needed for results.

By contrast, normal cells show very few CNV events, consistent with their annotations. 
Most intriguingly, unknown cells show CNV event signatures more similar to the malignant cells than to the normal cells, suggesting many of these cells may indeed be malignant. 

We also see traces of this relationship even when looking at the consensus cell types in conjunction with CNV events.
In Figure {@fig:fig5}D, we show the distributions of per-cell total CNV events for the most commonly-observed consensus cell types in the neuroblastoma library `SCPCL000130`.
Here, Unknown and neuron cells have distinctly higher total CNV values compared to other cell types, suggesting that they are likely to be malignant cells.
We see similar patterns for the ganglioglioma library `SCPCL000049` (Figure {@fig:figS4}B-C), where consensus T cells have low total CNV values, while other cell types including oligodendrocyte precursor cells, neuron associated cells, and Unknown cells have much higher total CNV values. 

> These observations provide confirmation that the identified consensus T cells (Figure {@fig:figS4}A) are indeed not likely to be malignant and provide evidence for which cells in the library may be malignant. 

Reviewer 2: ```suggestion
```
I don't think this sentence adds any information that hasn't already been stated and we mention this panel already, so I think you could remove it.
